canonicalwebteam/blog/wordpress_api.py

```diff
@@ -49,7 +46,7 @@ def get_articles(
     return process_response(response), total_pages
 
 
-def get_article(slug, tags=[], tags_exclude=[]):
+def get_article(slug="", tags=[], tags_exclude=[]):
```

Comment:
Why are you making the slug optional? Are you expecting to call `get_article(tags=['my-tag'])` or something, to get the first article of a tag? That would seem like a rather weird use of `get_article`.

I think it's best to leave `slug` as a mandatory argument, to signal that you're intended to get a specific article by slug.

Reply:
Yup! I fully agree